src/main/java/nextstep/subway/path/infra/DefaultLineFee.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.subway.path.infra;
+
+import nextstep.subway.common.Money;
+import nextstep.subway.line.domain.LineRepository;
+import nextstep.subway.path.domain.LineFee;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class DefaultLineFee implements LineFee {
+
+    private final LineRepository lineRepository;
+
+    public DefaultLineFee(final LineRepository lineRepository) {
+        this.lineRepository = lineRepository;
+    }
+
+    @Override
+    public Money settle(final List<Long> lineIds) {
+        return lineIds.stream()
```

Comment:
```suggestion
        return lineRepository.findAllById(lineIds)
        ...
```
라인 아이디를 여러개 받아오니 쿼리도 한 번에 조회하면 디비 조회 비용을 줄일 수 있겠네요